packages/atomic/src/utils/accessibility-utils.ts

```diff
@@ -68,6 +74,13 @@ export class FocusTargetController implements ReactiveController {
     this.host.addController(this);
   }
 
+  private clearFocusCallbacks() {
```

Comment:
We should adjust the unit tests for the `#focus` and `#disableForCurrentSearch` methods to account for the fact that focus callbacks should now be cleared.